src/js/views/scores.js

```diff
@@ -20,6 +20,18 @@ define('views/scores',[
                 $scope.rev = (String($scope.sort) === String(col)) ? !$scope.rev : !!defaultSort;
                 $scope.sort = col;
             };
+
+            $scope.sortIcon = function(col){
+                if(String($scope.sort)!== String(col)){//col and $scope.sort can be arrays, and so this is a quick and dirty way to check for equality
```

Comment:
We try not to do quick and dirty. There are better ways to compare them.
For instance, I found this here:
﻿https://gist.github.com/ain/8645033

Array.prototype.compare = function(array) {
  if (!array) {
    return false;
  }
  if(!(array instanceof Array)) {
     return this.length === 1 && this[0] === array;
  }
  if (this.length !== array.length) {
    return false;
  }
  for (var i = 0, l = this.length; i < l; i++) {
    if (this[i] instanceof Array && array[i] instanceof Array) {
      if (!this[i].compare(array[i])) {
        return false;
      }
    }
    else if (this[i] !== array[i]) {
      return false;
    }
  }
  return true;
}

And added the case for non-arrays.
I don't say this is the best solution, but there's no need to turn them into string just to compare them